lib/src/drawings.dart

```diff
@@ -85,8 +83,8 @@ class TextDrawing extends GYWDrawing {
   /// Creates a text element.
   const TextDrawing({
     required this.text,
-    this.font,
-    this.size,
+    required this.font,
+    required this.size,
```

Comment:
Sure.

Reply:
Should I also default the font to roboto mono normal?

Reply:
Acutally I can't. 

`The property 'font' can't be accessed on the type 'GYWFonts' in a constant expression.`

Too bad. It will stay required then.

Reply:
@TheToto42 I don't like that Dart doesn't allow you to use enum values in const contexts. Should I make font nullable and set if it's null, choose roboto mono by default?

Reply:
Yes, you can do that

Reply:
I solved this in a different way. If I let font be nullable, I would have to pay attention to make sure that null == roboto mono in the equality operator and the methods that use this font, and I didn't quite like that. I decided to work around the issue (https://github.com/dart-lang/language/issues/2374) by just defining a constant with the font, and the constructor can reference this constant instead of `GYWFonts.robotoMono.font`.

https://github.com/getyourway/flutter_gyw/blob/4b209fa0324cf2eee2247bdb94ef3cf2ff184947/lib/src/fonts.dart#L58-L70

Is this solution ok to you?